### PR TITLE
OGC submission rewording: Make clear it's a specification, not software

### PR DESCRIPTION
**Pull request by m-mohr**
Based on the feedback we got from OGC, I started rewording the OGC submission document.
The focus is to make more clear that we submit a specification (openEO) that consists of two sub-specifications (API + Processes) and that any software is not part of the submission. The software is just mentioned to prove adoption.

Please provide feedback via the GitHub PR review mechanism as we don't have a close next meeting scheduled. 
If we need a meeting to further discuss, we can set up an ad hoc meeting.

### CODE REVIEW

File: documents/ogc-submission.adoc [hunk 1]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
```
Comment (neteler) [inline documents/ogc-submission.adoc:47]: ... will be valid ...

File: documents/ogc-submission.adoc [hunk 2]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
 
 == Overview of proposed submission
 
-openEO defines an open application programming interface (API) that connects clients in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
+openEO specifies an open application programming interface (API) that connects clients in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
```
Comment (neteler) [inline documents/ogc-submission.adoc:50]:
```suggestion
openEO specifies an open application programming interface (API) that connects clients written in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
```

File: documents/ogc-submission.adoc [hunk 3]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
 
-The openEO specifications aim at increasing the interoperability of big EO data processing of satellite imagery in the cloud. Implementations of the openEO API can be used to add an interoperability layer on top of existing services.
-Multiple organizations agreed to define and implement such an API. Its development was driven by the need to overcome the challenges associated with different tools, APIs, and data formats in geospatial technology. openEO was developed bottom-up and each version was backed by implementations.
+The openEO specification aims at increasing the interoperability of big EO data processing of satellite imagery in the cloud. Implementations of openEO can be used to add an interoperability layer on top of existing services.
+Its development was driven by the need to overcome the challenges associated with different tools, APIs, and data formats in geospatial technology. openEO was developed bottom-up and each version of the specification was backed by implementations.
```
Comment (neteler) [inline documents/ogc-submission.adoc:53]:
```suggestion
Its development has been driven by the need to overcome the challenges associated with different tools, APIs, and data formats in geospatial technology. openEO was developed bottom-up and each version of the specification was backed by implementations.
```

File: documents/ogc-submission.adoc [hunk 4]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
 
-The primary use case for developing openEO was to simplify and unify the data processing using a common API specification with a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints supporting comparing and reproducing processing results more easily between different providers.
+The primary use case for specifying openEO was to simplify and unify the data processing using a common API and a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints supporting comparing and reproducing processing results more easily between different providers.
```
Comment (neteler) [inline documents/ogc-submission.adoc:55]:
```suggestion
The primary use case for specifying openEO was to simplify and unify the data processing using a common API and a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints, making it easier to compare and reproduce processing results between different providers.
```

File: documents/ogc-submission.adoc [hunk 5]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
 
-openEO is an open source project and the specification consists of two parts:
+The specification consists of two parts:
 * The openEO API, an HTTP API description specified as an OpenAPI document
 * The openEO Processes, a set of pre-defined processes defined following the openEO process description in JSON that is defined in the openEO API specification
```
Comment (neteler) [inline documents/ogc-submission.adoc:59]:
```suggestion
* The openEO Processes, a set of pre-defined processes defined following the openEO Process Description in JSON that is defined in the openEO API specification
```

File: documents/ogc-submission.adoc [hunk 6]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
 
-openEO is backed by various tools and resources that facilitate its use. These include a variety of server and client implementations, including a web interface for non-programmers and a server implementation that can run locally in a Python environment.
-
 == Relationship to other OGC Standards
 
 The openEO API builds on top of other standards and specifications and reuses other standards whenever possible.
 openEO's basis is HTTP with JSON as request and response encoding.
 The API specification reuses building blocks from the OGC API – Common Standard.
 openEO adheres to the STAC API specification, which is based on the OGC API – Features Standard.
 openEO allows exposure of a wide variety of visualization and download services using existing standards/specifications including OGC WMS, OGC WMTS, OGC WFS, OGC WCS, OGC API – Tiles, OGC API – Maps, OGC API – Features, and XYZ.
```
Comment (neteler) [inline documents/ogc-submission.adoc:71]:
```suggestion
openEO allows the exposure of a wide variety of visualization and download services using existing standards/specifications including OGC WMS, OGC WMTS, OGC WFS, OGC WCS, OGC API – Tiles, OGC API – Maps, OGC API – Features, and XYZ.
```

File: documents/ogc-submission.adoc (hunk 1 above)
Comment (m-mohr) [reply documents/ogc-submission.adoc:47]: That's originating from the OGC template, I'm not sure whether we should change that.

File: documents/ogc-submission.adoc (hunk 5 above)
Comment (m-mohr) [reply documents/ogc-submission.adoc:59]: I'm not sure, what do the others think?

File: documents/ogc-submission.adoc (hunk 6 above)
Comment (m-mohr) [reply documents/ogc-submission.adoc]: Thank you!

File: documents/ogc-submission.adoc (hunk 1 above)
Comment (neteler) [reply documents/ogc-submission.adoc:47]: Ah, right. So, let's scratch that one.

**Review by jdries — APPROVED** [review]
As it's my last day before summer leave, I'd like to approve already!

For evidence of adoption, if needed, I provide some paragraphs below. Feel free to pick what is suitable (if at all).

The European Space Agency (ESA) is adopting openEO as one of two options for future implementations of interoperable EO processing workflows and services. This adoption is happening in the frame of ESA APEx and Earth Code projects, and will take more concrete shape over the coming months.

While the exact wording of the APEx requirements document is still under review, the current version can already be found here:
https://esa-apex.github.io/apex_documentation/interoperability/algohostingenv.html#openeo-api-specific-requirements

While it is hard to obtain public numbers on overall uptake by users, some of the backends already report 300 active users on a monthly basis. This is already a substantial number considering that these are mainly researchers and developers working in earth observation processing. A further increase is expected as we see that backend implementations reach a maturity level that make them competitive with other well-known proprietary offerings for EO processing.

File: documents/ogc-submission.adoc [hunk 7]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
 
 == Overview of proposed submission
 
-openEO defines an open application programming interface (API) that connects clients in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
+openEO specifies an open application programming interface (API) that connects clients written in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
```
Comment (mkadunc) [inline documents/ogc-submission.adoc:50]:
```suggestion
openEO specifies an open application programming interface (API) for connecting applications and other client software to big Earth observation cloud back-ends in a simple and unified way.
```

File: documents/ogc-submission.adoc [hunk 8]
```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
 
-The primary use case for developing openEO was to simplify and unify the data processing using a common API specification with a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints supporting comparing and reproducing processing results more easily between different providers.
+The primary use case for specifying openEO was to simplify and unify the data processing using a common API and a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints, making it easier to compare and reproduce processing results between different providers.
```
Comment (mkadunc) [inline documents/ogc-submission.adoc:55]:
```suggestion
The primary use case for specifying openEO was to simplify and unify the data processing using a common API and a specification for a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints, making it easier to compare and reproduce processing results between different providers.
```

File: documents/ogc-submission.adoc (hunk 5 above)
Comment (mkadunc) [reply documents/ogc-submission.adoc:59]: I'm OK with lowercase - "process description" is neither a well-known name nor a term defined in this document.

```suggestion
* The openEO Processes, a set of requirements for pre-defined processes conforming to the openEO API specification
```

**Review by mkadunc — APPROVED** [review]

**Review by neteler — APPROVED** [review]

**Comment by m-mohr** [comment]
@aljacob @edzer @bschumac Any thoughts from your side?

**Comment by m-mohr** [comment]
I got a new template of the OGC submission document today.

I compared it to the old template and it has changes with regards to the implementations (chapter: "Evidence of implementation"), the adoption (chapter: "Information on adoption") and the motivation (chapter: "Motivation").

I restructured the document accordingly, but now I need your help and input:

1. For motivation, one or more supporting OGC members must describe why they are supporting this submission and how they feel it helps the geospatial community. I think we should provide it for at least 3 or more members. Who could write such a section?
2. We need to list details for each implementation. I started them for the clients and added some (unfilled) sections for server-side technology, but I generally need input. Everyone who is involved in any of these tools, please provide your input to the sections. You can also add new sections if you want to add another implementation. We don't need to list all implementations. I'm still waiting for a clarification that explains what "complete" means.
3. For Adoption, I took Jeroens text (thanks) and added download numbers for the R and Python client. I also reused the list of deployments we had and added Web Editor deployments. If anyone has additional information, please add it.

File: documents/ogc-submission.adoc [hunk 9]
```diff
@@ -47,38 +47,57 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
 The project is governed by a Project Steering Committee: https://openeo.org/psc/
 
-openEO is backed by various tools and resources that facilitate its use. These include a variety of server and client implementations, including a web interface for non-programmers and a server implementation that can run locally in a Python environment.
+== Motivation of supporting OGC Members for this submission
+
+////
+Provide a brief paragraph from one or more supporting OGC members that describes why they are supporting this submission and how they feel it helps the geospatial community.
+////
+
+...
```
Comment (m-mohr) [inline documents/ogc-submission.adoc:75]:
```suggestion
...

=== European Space Agency (ESA)

openEO provides valuable standardisation and abstraction of EO processing and analysis chains across different federated compute environements. The later point is especially important in the fragmented European cloud landscape. The openEO specification has been adopted by various organisations, partners or stakeholders of ESA (e.g. EC JRC, EURAC, INPE, various academic labs) and companies (e.g. VITO, Sinergise, EODC, EOX, eGEOS, Mundialis, Google Earth Engine). ESA has further invested on the development of the operational openEO platform service ([https://openeo.cloud/)](https://openeo.cloud/).) based on the openEO specification. Thanks to this, the Copernicus Data Space Ecosystem (CDSE), sucessor to the Copernicus DIAS initive, jointly implemented/managed by ESA and the European Comission, has selected the openEO specification as a core API to enable the community to process EO data and collaborate on workflows and analytics. ESA is also investing on the OGC Geodatacube API standardization process of which openEO is considered as one of the key standards. Finally at ESA various efforts on operationalising EO science and aplication results are building on openEO due to its FAIR compliant properties, this is also why the evolution of the EO Exploitation Platform Common Architecture is going to support openEO. Making openEO an OGC Community standard is a key step to foster more widespread adoption in industry and public domain and allow to extend its current reach beyond Europe.

```

File: documents/ogc-submission.adoc [hunk 10]
```diff
@@ -47,38 +47,64 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
...rs or stakeholders of ESA (e.g. EC JRC, EURAC, INPE, various academic labs) and companies (e.g. VITO, Sinergise, EODC, EOX, eGEOS, Mundialis, Google Earth Engine). ESA has further invested on the development of the operational openEO platform service ([https://openeo.cloud/)](https://openeo.cloud/).) based on the openEO specification. Thanks to this, the Copernicus Data Space Ecosystem (CDSE), sucessor to the Copernicus DIAS initive, jointly implemented/managed by ESA and the European Comission, has selected the openEO specification as a core API to enable the community to process EO data and collaborate on workflows and analytics. ESA is also investing on the OGC Geodatacube API standardization process of which openEO is considered as one of the key standards. Finally at ESA various efforts on operationalising EO science and aplication results are building on openEO due to its FAIR compliant properties, this is also why the evolution of the EO Exploitation Platform Common Architecture is going to support openEO. Making openEO an OGC Community standard is a key step to foster more widespread adoption in industry and public domain and allow to extend its current reach beyond Europe.
```
Comment (neteler) [inline documents/ogc-submission.adoc:79]:
```suggestion
openEO provides valuable standardisation and abstraction of EO processing and analysis chains across different federated compute environments. The latter point is especially important in the fragmented European cloud landscape. The openEO specification has been adopted by various organisations, partners or stakeholders of ESA (e.g. EC JRC, EURAC, INPE, various academic labs) and companies (e.g. VITO, Sinergise, EODC, EOX, eGEOS, mundialis, Google Earth Engine). ESA has also invested in the development of the operational openEO platform service ([https://openeo.cloud/](https://openeo.cloud/)) based on the openEO specification. Thanks to this, the Copernicus Data Space Ecosystem (CDSE), successor to the Copernicus DIAS initiative, jointly implemented and managed by ESA and the European Commission, has selected the openEO specification as a core API to enable the community to process EO data and collaborate on workflows and analytics. ESA is also investing on the OGC Geodatacube API standardization process of which openEO is considered as one of the key standards. Finally at ESA various efforts to operationalise EO science and application results are building on openEO due to its FAIR compliant properties, which is why the evolution of the EO Exploitation Platform Common Architecture will also support openEO. Making openEO an OGC Community standard is a key step to foster more widespread adoption in industry and public domain and allow to extend its current reach beyond Europe.
```

Comment (neteler) [reply documents/ogc-submission.adoc:79]: (fixed typos and grammar)

File: documents/ogc-submission.adoc [hunk 11]
```diff
@@ -47,38 +47,71 @@ Once approved, the Community Standard Work Item defined by this document is vali
...
...teroperability for spatial data. It took the initiativ for the initial 2017 H2020 project that started openEO. The reason for this was that at that time, cloud platforms for processing Earth Observation data were not interoperable, which led to platform lock-in and made it hard to compare results for the same process obtained from two different platforms. Many such platforms were based on closed source software. Difficulty to compare cloud platforms conflicts with principles of open science and hinders reproducibility and research validation. The openEO API resolved this problem by providing a single, uniform API to a variety of back-end cloud platforms. In addition, it made it easier to develop clients in different data science languages or environments, which in turn made it easier for end-users to switch languages, or to compare developments carried out in different data science languages. The currently existing and evolving ecosystem of clients and back-ends using the openEO encourages the development of new analysis methods, and has led to an increase in communications between and among researchers and/or research software engineers that previously had very little interaction.
```
Comment (m-mohr) [inline documents/ogc-submission.adoc:89]:
```suggestion
The Institute for Geoinformatics has a 30 year history of research in interoperability for spatial data. It took the initiative for the initial 2017 H2020 project that started openEO. The reason for this was that at that time, cloud platforms for processing Earth Observation data were not interoperable, which led to platform lock-in and made it hard to compare results for the same process obtained from two different platforms. Many such platforms were based on closed source software. Difficulty to compare cloud platforms conflicts with principles of open science and hinders reproducibility and research validation. The openEO API resolved this problem by providing a single, uniform API to a variety of back-end cloud platforms. In addition, it made it easier to develop clients in different data science languages or environments, which in turn made it easier for end-users to switch languages, or to compare developments carried out in different data science languages. The currently existing and evolving ecosystem of clients and back-ends using the openEO encourages the development of new analysis methods, and has led to an increase in communications between and among researchers and/or research software engineers that previously had very little interaction.
```

**Pull request merged**